LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift

```diff
@@ -36,14 +37,15 @@ public struct CorrectionRangeOverridesEditor: View {
     @Environment(\.dismiss) var dismiss
     @Environment(\.authenticate) var authenticate
 
-    public init(
+    fileprivate init(
```

Comment:
[nit] curious why this init is `fileprivate`

Reply:
My philosophy for protection levels is everything should be the most protected that it can be.  This Editor is really only useful when given a `TherapySettingsViewModel`.  This `init` is, TBH, the "old" init that was reused when we introduced the view model.  It was never removed...this is a nod in the direction of getting rid of it.